cricPredict/Templates/create_group.html

```diff
@@ -26,8 +26,22 @@ <h2>Create Group</h2>
                 </select>
               </div>
 
-               <input type="number" name="admin_id" id="admin_id" hidden value=1>
+               <input type="number" name="admin" id="admin" hidden value=1>
```

Comment:
I thought we removed this from the frontend

Reply:
I forgot to push the updated code